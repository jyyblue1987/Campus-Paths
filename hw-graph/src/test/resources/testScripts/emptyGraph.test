# To test if we can create a graph

# Create a graph
CreateGraph A

# Print out the nodes in the graph
ListNodes A