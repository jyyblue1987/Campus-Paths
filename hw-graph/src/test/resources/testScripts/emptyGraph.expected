# To test if we can create a graph

# Create a graph
created graph A

# Print out the nodes in the graph
A contains: